### ISSUE TITLE
Authentication bugs

### ISSUE THREAD

**Issue by kajchang** [petcodeapp/web#9]
- models aren't serializable
- vaccinations aren't loaded as models
- `get` and `getClient` in models don't specify a collection

**Comment by rishiosaur** [comment]
@kajchang on that first point—every big model exposes an `asInterface` method that's serializable

**Comment by kajchang** [comment]
yeah, I used `asInterface` but Next still doesn't like it because it's a class, only accepts straight JSON for SSR props
